src/test/java/org/folio/rest/impl/EHoldingsProxyTypesImplTest.java

```diff
@@ -57,14 +57,6 @@ public void shouldReturnForbiddenWhenRMAPIRequestCompletesWith403ErrorStatus() {
     getResponse(EHOLDINGS_PROXY_TYPES_URL).statusCode(HttpStatus.SC_FORBIDDEN);
   }
 
-  private ExtractableResponse<Response> getResponseWithStatus(String resourcePath, int expectedStatus) {
```

Comment:
Why is this test removed as part of this PR?

Reply:
Moved method to base class